todo_app/data/db_items.py

```diff
@@ -0,0 +1,87 @@
+from flask import session
+import pymongo
+import os
+from  todo_app.data.Item import Item
+from datetime import datetime
+
+
+max_id = 0
+
+def getDb():
+    client = pymongo.MongoClient(os.getenv('AZ_MONGODB_CONNECTION'))
+    db = client[os.getenv('AZ_MONGODB_DB_NAME')]
+    return db
+
+def get_items():
+    """
+    Fetches items from the connected DB.
+
+    Returns:
+        list: The list of saved items.
+    """     
+    db = getDb()
+
+    mdb_items_col = db.items
+    mdb_items = mdb_items_col.find({})
+    
+    cards = []
+
+    global max_id
```

Comment:
Why do you want a short id? It feels like you're having to write a fair amount of logic to handle this, when we could just rely on the auto-generated id, which will save us having to maintain this in future behaviour

I think we can get rid of all of the references to this, and just use the inbuilt `_id` field to simplify this fairly drastically e.g.:
```
    for it in mdb_items:
        cards.append(Item(str(it['_id']), it['name'], it['status']))
```

This mostly "just works" though there are a couple of places we no longer need to cast the ids to an int. We might then want to not show the ids to a user, but that should be fine

Reply:
yes, absolutely, I didn't know I could use _id directly